dockerfiles/compilation/java-11/compile.sh

```diff
@@ -73,16 +73,16 @@ unzip $GLOW_ROOT_ZIP
 if [ $? -eq 0 ]; then
    if [ -d "$CASSANDRA_RABBITMQ_LDAP_DESTINATION" ]; then
       echo "Copying cassandra - rabbitMQ - Ldap JARs"
-      cp server/container/guice/cassandra-rabbitmq-ldap-guice/target/james-server-cassandra-rabbitmq-ldap-guice.jar $CASSANDRA_RABBITMQ_LDAP_DESTINATION || true
-      cp -r server/container/guice/cassandra-rabbitmq-ldap-guice/target/james-server-cassandra-rabbitmq-ldap-guice.lib $CASSANDRA_RABBITMQ_LDAP_DESTINATION || true
+      cp server/container/guice/cassandra-rabbitmq-guice/target/james-server-cassandra-rabbitmq-guice.jar $CASSANDRA_RABBITMQ_LDAP_DESTINATION || true
+      cp -r server/container/guice/cassandra-rabbitmq-guice/target/james-server-cassandra-rabbitmq-guice.lib $CASSANDRA_RABBITMQ_LDAP_DESTINATION || true
```

Comment:
I don't think you want to change that. Seeing that you are basing your module chooser on whether or not you have an ldap conf in usersrepository.xml, if you point the build  towards the conf of `cassandra-rabbitmq-guice`, you will not end up with a ldap support server here. You will likely generate twice the same cassandra-rabbitmq-guice image here, right?

Reply:
Here we only copy binaries, not the conf.

The conf comes from the dockerfiles folder. We should be good.

Reply:
You are right I mixed things up here